shell/platform/embedder/flutter_engine_exports.lst

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Linker script that hides symbols with the prefix "FT_" so that global freetype
+# symbols referenced by Skia are not overriden by an external shared library at
+# runtime.
+
+{
+  global:
+    *;
+  local:
+    FT_*;
```

Comment:
First of all, thank you for your hard work. I've reviewed the things that you've tried to solve this problem.
And I'm worried that other libs may also have similar problems. 
I'm not sure about this, what about restrict the global symbols of libflutter_tizen and lib flutter plugins? Will it work?

Reply:
> `libflutter_tizen.so` or `libflutter_plugins.so` does not have static dependencies on external libs

Um.. I'm worried about some problem related to other libraries statically linked to flutter-engine. Their symbols can also be misresolved on the same principle? not only freetype.

In Android shell, it seems that all symbols are made local and selectively exposed to global. but I'm not sure that is the same purpose that you do. I think you are the most expert on this. I'm just curious about your opinion.

Reply:
> Their symbols can also be misresolved on the same principle? not only freetype.

Yes, but I don't know the exact list of symbols that `libflutter_engine.so` should export and it's not easy to figure it out.

> In Android shell, it seems that all symbols are made local and selectively exposed to global.

The android shell works very differently from ours. It has a Java wrapper (based on JNI) for all the flutter APIs and therefore has to only export `JNI_OnLoad`.

Reply:
All right, let's move on as it is